qcelemental/tests/test_molecule.py

```diff
@@ -359,3 +360,41 @@ def test_fragment_charge_configurations(f1c, f1m, f2c, f2m, tc, tm):
 
     assert pytest.approx(mol.get_fragment([1], 0).molecular_charge) == f2c
     assert mol.get_fragment(1, [0]).molecular_multiplicity == f2m
+
+
+def test_nre_nelec():
```

Comment:
Maybe `nuclear_repulsion_and_nelec` or something? `nre` is a shorthand too far I think.